docs/usage.md

```diff
@@ -62,6 +62,57 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Handling git branches
```

Comment:
I found how this section is broken up confusing. It seems to be using the subcommand structure to inform the organization rather than use cases. I think it would be much easier to consume if it was organized by use-case. So instead of having a section "Handling git branches", there would be a section, "Creating a branch". Then a completely different section "Updating your existing branch".